SportsHub/SportsHub.Application/Authentication/Models/Options/PasswordHashingOptions.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SportsHub.AppService.Authentication.Models.Options
+{
+    public sealed class PasswordHashingOptions
+    {
+        public int Iterations { get; set; } = 10000;
```

Comment:
Do you plan to change the value of this property in the application during runtime?

Reply:
Not for now.

Reply:
In such case, you can remove `set`